src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1738,6 +1764,18 @@ public void Error(string p)
             ToolTipText = p;
         }
 
+        /// <summary>
+        /// Set an info on a node.
+        /// </summary>
+        /// <param name="p">The info text.</param>
+        public void Info(string p)
+        {
+            State = ElementState.Info;
+            infos.Add(new Info(p, ElementState.Info));
+
+            ToolTipText = p;
```

Comment:
this is deprecated, it's no longer required to set this.

Reply:
Can you use the existing constructor instead of adding a new one?
```
internal Info(string message, ElementState state = ElementState.Active)
        {
            Message = message;
            State =  state;
        }
```

Reply:
I have added the Info() to NodeModel class to be consistent with the Error() and Warning(). I thought it would be easy for developers to just call that method on NodeModel class. What do you think?

I have removed the following statement.

> ToolTipText = p;